client/components/forms/docs/example.jsx

```diff
@@ -35,7 +36,10 @@ import PhoneInput from 'components/phone-input';
 /**
  * Internal dependencies
  */
-var countriesList = require( 'lib/countries-list' ).forSms();
+const countriesList = require( 'lib/countries-list' ).forSms();
+const currencies = require( 'lib/format-currency/currencies' ).CURRENCIES
```

Comment:
OK, it seems then that the only reason to use `require` here was convenience -- `require` call can be used as an expression.

I did the migration to `import` in #17427, which already touches this code.